crates/wasi-http/Cargo.toml

```diff
@@ -31,9 +31,9 @@ wasmtime = { workspace = true, features = ['component-model'] }
 
 # The `ring` crate, used to implement TLS, does not build on riscv64 or s390x
 [target.'cfg(not(any(target_arch = "riscv64", target_arch = "s390x")))'.dependencies]
```

Comment:
For this I might recommend adding a commit with `prtest:full` somewhere in the commit message to ensure that this can get past CI. I vaguely recall that rustls and/or ring and/or something doesn't compile on riscv64/s390x, hence the conditional here. That'll make the integration of `wasmtime-wasi-tls` into the CLI a bit hairier if so, but probably still workable.

Reply:
made some updates to address this.  but now getting `register mapping is currently only implemented for x86_64` on some of the other platforms which I were working previously.  I am not sure what this means yet, as I wouldn't have expected the changes to make a difference.

Reply:
Looked a bit more at this. It wasn't passing prior to my changes.  I am not sure what causes this, if it is expected, or how to debug it.

@alexcrichton could you give me a pointer to what might be going wrong here?

Reply:
Ah that's a bad error, but it bottoms out in `config.debug_info(true);` which isn't supported on Pulley. I'll go try and improve that error.

Reply:
Thanks! Removing that line fixed the issue.